templates/expo-template-bare-minimum/ios/Podfile

```
@@ -22,6 +22,16 @@ target 'HelloWorld' do
   use_frameworks! :linkage => podfile_properties['ios.useFrameworks'].to_sym if podfile_properties['ios.useFrameworks']
   use_frameworks! :linkage => ENV['USE_FRAMEWORKS'].to_sym if ENV['USE_FRAMEWORKS']
 
+  # Flipper support in expo via build properties plugin
+  expo_flipper = FlipperConfiguration.disabled
+  if ENV['FLIPPER_DISABLE'] != "1" && ENV['NO_FLIPPER'] != "1" && defined?(podfile_properties['ios.flipper']) then
+    if podfile_properties['ios.flipper'] == true then
```

```suggestion
    if podfile_properties['ios.flipper'] == "true" then
```

a pitfall for expo-build-properties plugin on ios is that it generates value in string, so you should use `"true"` or `"false"` 😅

---

Yeah, caught that too. Strings all the way through. Fixup includes both `'true'` and `'false'` based on the pattern from `newArchEnabled`